fcrepo-jms/src/test/java/org/fcrepo/jms/headers/DefaultMessageFactoryTest.java

```diff
@@ -96,6 +101,38 @@ public void testBuildMessageContent() throws RepositoryException, JMSException {
         assertEquals("Got wrong identifier in message!", testPath, msg.getStringProperty(IDENTIFIER_HEADER_NAME));
     }
 
+    @Test
+    public void testBuildFixityMessageContent() throws RepositoryException, JMSException {
+        when(mockFixityEvent.getDate()).thenReturn(1428335457057L);
+        when(mockFixityEvent.getType()).thenReturn(128);
+        when(mockFixityEvent.getPath()).thenReturn("/83/0b/57/17/830b5717-1434-4653-af9c-a00d6d020426");
+        when(mockFixityEvent.getIdentifier()).thenReturn("/83/0b/57/17/830b5717-1434-4653-af9c-a00d6d020426");
+        when(mockFixityEvent.getBaseURL()).thenReturn("http://localhost:8080/rest/");
+        final HashMap<String,String> info = new HashMap<String,String>();
+        info.put("http://www.loc.gov/premis/rdf/v1#hasMessageDigest",
+                "urn:sha1:ca063d541865ab93d062f35800feb5011183abe6");
+        info.put("http://www.loc.gov/premis/rdf/v1#hasSize", "56114^^http://www.w3.org/2001/XMLSchema#long");
+        info.put("http://fedora.info/definitions/v4/repository#status", "SUCCESS");
```

Comment:
The `repository:status` property has been removed. See https://github.com/fcrepo4/ontology/commit/88766107f78a5d7fd1ebc3044142ea9034617e61

Wouldn't the `premis:hasEventOutcome` property be a better fit?